src/pathogen_embed/pathogen_embed.py

```diff
@@ -296,18 +259,6 @@ def embed(args):
             "n_init": 2
         }
 
-    # Override defaults with parameter values passed through cluster data, if
-    # possible.
-    if cluster_data is not None and args.command != "pca":
-        for key, value in cluster_data.items():
-            if key in embedding_parameters:
-                value_type = type(embedding_parameters[key])
-                print(
-                    f"INFO: Replacing embedding parameter {key} value of '{embedding_parameters[key]}' with '{value_type(value)}' provided by '{args.cluster_data}'.",
-                    file=sys.stderr
-                )
-                embedding_parameters[key] = value_type(value)
-
```

Comment:
Related to the comment above about keeping a `--cluster-data`-style argument for this command, we need to keep this block of logic, so we can override default parameters with a single file from the command line.